AndroidAnnotations/androidannotations-api/src/main/java/org/androidannotations/annotations/PreferenceChange.java

```diff
@@ -42,8 +42,13 @@
  * <li>A {@link android.preference.Preference Preference} parameter to know
  * which preference was targeted by this event</li>
  * <li>An {@link Object} or {@link java.util.Set Set of strings} or
- * {@link Boolean} or {@link String} to obtain the new value of the
- * {@link android.preference.Preference Preference}</li>
+ * {@link Boolean} or {@link Float} or {@link Integer} or {@link Long} or
```

Comment:
not sure how it is in english, but in german we would comma seperate the words but the last. also personally i would change the order a little bit

> An Object, String, Set of Strings and also Boolean, Float, Integer, Long or thier corresponding primitive types